platform-docs/docs/intro.md

```diff
@@ -28,58 +27,73 @@ To build a block editor, you need to install the following dependencies:
+```html
+<link href="node_modules/@wordpress/components/build-style/style.css" rel="stylesheet" vite-ignore/>
+<link href="node_modules/@wordpress/block-editor/build-style/style.css" rel="stylesheet" vite-ignore/>
+```
+:::note
+
+There are more styles needed but can’t be added here because they are for the editor’s content which is in a separate document within an `<iframe>`. We’ll add those styles via the `BlockCanvas` component in a later step.
+
+:::
+
+Next, we’ll add blocks for the editor to work with. In `src/main.jsx` import and call `registerCoreBlocks`:
+```js
+import { registerCoreBlocks } from '@wordpress/block-library'
+registerCoreBlocks();
+```
+
+Finally, we’ll put our editor together. In `src/App.jsx` replace the contents with the following code:
 
- - Update your `index.jsx` file with the following code:
 ```jsx
-import { createElement, useState } from "react";
-import { createRoot } from 'react-dom/client';
+import { useState } from "react";
 import {
   BlockEditorProvider,
   BlockCanvas,
 } from "@wordpress/block-editor";
-import { registerCoreBlocks } from "@wordpress/block-library";
 
-// Default styles that are needed for the editor.
-import "@wordpress/components/build-style/style.css";
-import "@wordpress/block-editor/build-style/style.css";
-
-// Default styles that are needed for the core blocks.
-import "@wordpress/block-library/build-style/common.css";
-import "@wordpress/block-library/build-style/style.css";
-import "@wordpress/block-library/build-style/editor.css";
-
-// Register the default core block types.
-registerCoreBlocks();
+// Base styles for the content within the block canvas iframe.
+import componentsStyles from "@wordpress/components/build-style/style.css?raw";
+import blockEditorContentStyles from "@wordpress/block-editor/build-style/content.css?raw";
+import blocksStyles from "@wordpress/block-library/build-style/style.css?raw";
+import blocksEditorStyles from "@wordpress/block-library/build-style/editor.css?raw";
 
-function Editor() {
-  const [blocks, setBlocks] = useState([]);
+export default function Editor() {
+  const [ blocks, setBlocks ] = useState( [] );
   return (
     /*
       The BlockEditorProvider is the wrapper of the block editor's state.
       All the UI elements of the block editor need to be rendered within this provider.
     */
     <BlockEditorProvider
-      value={blocks}
-      onChange={setBlocks}
-      onInput={setBlocks}
+      value={ blocks }
+      onChange={ setBlocks }
+      onInput={ setBlocks }
     >
-      {/*
-          The BlockCanvas component renders the block list within an iframe
-          and wire up all the necessary events to make the block editor work.
-        */}
-      <BlockCanvas height="500px" />
+      { /*
+        The BlockCanvas component renders the block list within an iframe
+        and wires up all the necessary events to make the block editor work.
+      */ }
+      <BlockCanvas
+        height="500px"
+        styles={ [
```

Comment:
Do you think there's any performance value in putting a stable array here (maybe define this array outside the component)